src/framework/mlt.vers

```diff
@@ -647,4 +647,10 @@ MLT_7.10.0 {
     mlt_image_fill_checkerboard;
     mlt_image_fill_white;
     mlt_image_rgba_opaque;
+    mlt_property_set_color;
+    mlt_property_get_color;
+    mlt_property_anim_set_color;
+    mlt_property_anim_get_color;
+    mlt_properties_anim_set_color;
+    mlt_properties_anim_get_color;
```

Comment:
These symbols now need to moved to a new block with name `MLT_7.12.0`.

Reply:
Done